.github/workflows/master.yml

```diff
@@ -247,6 +247,14 @@ jobs:
       build_name: binary_s390x
       data: ${{ needs.RunConfig.outputs.data }}
       checkout_depth: 0
+  BuilderBinLOONGARCH64:
```

Comment:
Cosmetic: Could we rename that to `BuilderBinLoongarch64`?

Reply:
Yes.